src/com/ar4android/AR4AndroidApplication.java

```diff
@@ -1,83 +1,157 @@
+		// Do not display statistics
 		setDisplayStatView(false);
 		setDisplayFps(false);
-		// We use custom viewports - so the main viewport does not need to contain the rootNode
+		// we use our custom viewports - so the main viewport does not need the  rootNode
 		viewPort.detachScene(rootNode);
 		initVideoBackground(settings.getWidth(), settings.getHeight());
+		initForegroundScene();	
+		initBackgroundCamera();		
+		initForegroundCamera(mForegroundCamFOVY);
 	}
 
 	// This function creates the geometry, the viewport and the virtual camera
 	// needed for rendering the incoming Android camera frames in the scene
 	// graph
 	public void initVideoBackground(int screenWidth, int screenHeight) {
-		// Create a Quad shape. 
+		// Create a Quad shape.
 		Quad videoBGQuad = new Quad(1, 1, true);
 		// Create a Geometry with the Quad shape
-		mVideoBGGeom = new Geometry("quad", videoBGQuad);		 
-		float newWidth = 1.f * screenWidth / screenHeight;		
+		mVideoBGGeom = new Geometry("quad", videoBGQuad);
+		float newWidth = 1.f * screenWidth / screenHeight;
 		// Center the Geometry in the middle of the screen.
 		mVideoBGGeom.setLocalTranslation(-0.5f * newWidth, -0.5f, 0.f);//
-		// Scale (stretch) the width of the Geometry to cover the whole screen width.
+		// Scale (stretch) the width of the Geometry to cover the whole screen
+		// width.
 		mVideoBGGeom.setLocalScale(1.f * newWidth, 1.f, 1);
 		// Apply a unshaded material which we will use for texturing.
-		mvideoBGMat = new Material(assetManager, "Common/MatDefs/Misc/Unshaded.j3md");
+		mvideoBGMat = new Material(assetManager,
+				"Common/MatDefs/Misc/Unshaded.j3md");
 		mVideoBGGeom.setMaterial(mvideoBGMat);
-		// Create a new texture which will hold the Android camera preview frame pixels.
+		// Create a new texture which will hold the Android camera preview frame
+		// pixels.
 		mCameraTexture = new Texture2D();
+
 		
+		mSceneInitialized = true;
+	}
+	
+	public void initBackgroundCamera() {
 		// Create a custom virtual camera with orthographic projection
-		Camera videoBGCam = cam.clone();
+		Camera videoBGCam = cam.clone();		
 		videoBGCam.setParallelProjection(true);
-		// Also create a custom viewport. 
+		// Also create a custom viewport.
 		ViewPort videoBGVP = renderManager.createMainView("VideoBGView",
 				videoBGCam);
-		// Attach the geometry representing the video background to the viewport.
+		// Attach the geometry representing the video background to the
+		// viewport.
 		videoBGVP.attachScene(mVideoBGGeom);
-		mSceneInitialized = true;		
 	}
+	public void initForegroundScene() {
+		// Load a model from test_data (OgreXML + material + texture)
+        Spatial ninja = assetManager.loadModel("Models/Ninja/Ninja.mesh.xml");
+        ninja.scale(0.025f, 0.025f, 0.025f);
+        ninja.rotate(0.0f, -3.0f, 0.0f);
+        ninja.setLocalTranslation(0.0f, -2.5f, 0.0f);
+        rootNode.attachChild(ninja);
+        
+        // You must add a light to make the model visible
```

Comment:
To make the model visible, add a directional light shining from the top front onto the model. (can try not adding the light and see the result).